stdlib/@tests/test_cases/builtins/check_pow.py

```diff
@@ -47,7 +47,9 @@
 assert_type(complex(6) ** 6.2, complex)
 assert_type(pow(complex(9), 7.3, None), complex)
 
-assert_type(pow(Fraction(), 4, None), Fraction)
+# pyright infers Fraction | float | complex, while mypy infers Fraction.
+# This is probably because of differences in @overload handling.
+# assert_type(pow(Fraction(), 4, None), Fraction)
```

Comment:
A small suggestion, we can still check for our mypy expectation, *and* see if either type checker change their overload handling in a way that affects this
```suggestion
assert_type(pow(Fraction(), 4, None), Fraction)  # pyright: ignore[reportAssertTypeFailure]
```

imo your comment is already enough to explain the presence of this suppression comment.